packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -142,10 +142,11 @@ class FlutterDevice {
         extraFrontEndOptions: extraFrontEndOptions,
         platformDill: globals.fs.file(platformDillPath).absolute.uri.toString(),
         dartDefines: buildInfo.dartDefines,
-        librariesSpec: globals.fs
-            .file(globals.artifacts!.getHostArtifact(HostArtifact.flutterWebLibrariesJson))
-            .uri
-            .toString(),
+        librariesSpec:
```

Comment:
I think you may have accidentally formatted these files with an older version of the Dart formatter. Can you upgrade your Flutter installation to the latest stable and format this again?